src/odemis/acq/stitching/_registrar.py

```diff
@@ -487,7 +489,8 @@ def addTile(self, tile, dependent_tiles=None):
 
     def getPositions(self):
         """
-        Returns the registered positions.
+        Updates the registered positions (found using cross correlation and a min spanning tree) and returns the 
+        adjusted tile_positions & dep_tile_positions.
```

Comment:
Please add a line about `.registered_positions` being updated when this function is called.